api-spec/relayapi_auto_content_topic.yaml

```diff
@@ -1,7 +1,7 @@
 # /relay/v1/auto/messages/{contentTopic}:  # Note the plural in messages
 get: # get_waku_v2_relay_v1_auto_messages
   summary: Get the latest messages on the polled topic
-  description: Get a list of messages that were received on a subscribed Content topic after the last time this method was called.
+  description: Get a list of messages that were received on a subscribed Content topic after the last time this method was called. Messages are selected from the last 50 messages received by the node.
```

Comment:
For relay it is not exactly true.
For relay we have a cli arg `--rest-relay-cache-capacity` which defaults to 30 and that is applied.
...but configurable on demand.